css/style.css

```diff
@@ -1,63 +1,4 @@
 /* General page styles */
 
```

Comment:
Why is this file clean? 🤔

Reply:
in my work i only used the home.css, i dont touch other files pull from develop.